web-ts/src/components/__tests__/FeedbackBoxContainer.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { fireEvent, render, waitFor } from '@testing-library/react'
+import {
+  CATEGORIES_FEEDBACK_TYPE,
+  EVENTS_FEEDBACK_TYPE,
+  OFFER_FEEDBACK_TYPE,
+  OFFERS_FEEDBACK_TYPE,
+  PAGE_FEEDBACK_TYPE,
+  SPRUNGBRETT_OFFER
+} from 'api-client'
+import { ThemeProvider } from 'styled-components'
+import { Routes } from '../../routes/App'
+import { lightTheme } from 'build-configs/integreat/theme'
+import FeedbackBoxContainer from '../FeedbackBoxContainer'
+import { SendingStatusType } from '../FeedbackModal'
+
+const mockRequest = jest.fn()
+jest.mock('react-i18next')
+jest.mock('api-client', () => {
+  return {
+    ...jest.requireActual('api-client'),
+    createFeedbackEndpoint: (baseUrl: string) => ({
+      request: mockRequest
+    })
+  }
+})
+
+// TODO reenable after route setup is finished
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('FeedbackBoxContainer', () => {
```

Comment:
It makes no sense to test this component until the Routing to Feedback is not yet setup correctly (especially as we previously relied on the redux route payload)